src/components/Chat/hooks/useIsOnline.ts

```diff
@@ -44,9 +41,9 @@ export const useIsOnline = <
 
     const handleRecoveredEvent = () => setConnectionRecovering(false);
 
-    const notifyChatClient = (netInfoState: NetInfoState) => {
-      if (client && client.wsConnection) {
-        if (netInfoState?.isConnected) {
+    const notifyChatClient = (netInfoState: boolean) => {
+      if (client?.wsConnection) {
+        if (netInfoState) {
```

Comment:
so something changed here?

Reply:
yeah so this is passed in from the `native-package` stuff and then into `src/native.ts` and if you look at how it's returned there, it's sending back a boolean